skynet/modules/stt/streaming_whisper/meeting_connection.py

```diff
@@ -31,12 +32,14 @@ def __init__(self, ws: WebSocket):
         self.meeting_language = None
         self.tokenizer = None
 
-    async def update_initial_prompt(self, new_transcription: str):
-        self.previous_transcription_store.append(self.tokenizer.encode(f' {new_transcription.strip()}'))
-        if len(self.previous_transcription_tokens) > max_finals:
-            self.previous_transcription_store.pop(0)
-        # flatten the list of lists
-        self.previous_transcription_tokens = list(chain.from_iterable(self.previous_transcription_store))
+    async def update_initial_prompt(self, payloads: list[TranscriptionResponse]):
```

Comment:
Since payloads is a list of responses, I assume it's from previous transcriptions? If so, can we call it previous_payloads, for clarity?